### PR TITLE
Fixed to show spinner on data load

### PR DESCRIPTION
**Pull request by stephanwlee**
Also fixed the issue where, when showing the spinner, the iron-collapse
does not always open. Note that this collapse bug did exist in version
1,10 too,

### CODE REVIEW

File: tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts [hunk 1]
```diff
@@ -120,7 +119,8 @@ export const DataLoaderBehavior = {
 
     if (!this.isAttached) return;
     this._loadDataAsync = this.async(() => {
-      this.dataLoading = true;
+      // Using a setter does not notify the impl observers.
```
Comment (nfelt) [inline tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts]: Hmm, this is basically the same issue as the loadKey issue, right?  So it seems like maybe the behavior observer/property connection is buggy or broken somehow.

File: tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts [hunk 2]
```diff
@@ -140,7 +140,7 @@ export const DataLoaderBehavior = {
       });
 
       return Promise.all(promises).then(this._canceller.cancellable(result => {
-        this.dataLoading = false;
+        this.set('dataLoading', false);
```
Comment (nfelt) [inline tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts]: Can you add the same comment here?

File: tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts [hunk 3]
```diff
@@ -58,7 +58,6 @@ export const DataLoaderBehavior = {
 
     dataLoading: {
       type: Boolean,
-      readOnly: true,
```
Comment (nfelt) [inline tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts]: Does this have to be non-readOnly in order to use set()?  It's still logically readonly though, right?

Comment (stephanwlee) [reply tensorboard/components/tf_line_chart_data_loader/data-loader-behavior.ts]: Ah, you were correct! I always forget that you ned to use a special setter when setting a value to a readOnly property. I changed other codes to use `this._setDataLoading` and kept this readOnly. Thanks!

**Pull request merged**